### PR TITLE
PR for #25: Add structure to input folder for tests

### PR DESCRIPTION
**Pull request by santiagohermo**
In this branch we implement some fixes flagged by @SimonFreyaldenhoven in [this comment](https://github.com/JMSLab/eventstudyr/pull/24#issuecomment-1474350312). This is in anticipation for the initial release in #25.

Who has some time for a quick review @jmshapir @nateschor @SimonFreyaldenhoven? It just some fancier writing and execution of the existing code, it shouldn't take too long.

### CODE REVIEW

File: tests/testthat/input/code/build.do
[the remark this answers is not in the document]
Comment (santiagohermo) [reply tests/testthat/input/code/build.do:1]: Makes sense! Implemented in https://github.com/JMSLab/eventstudyr/pull/26/commits/ecd046fcb6db37e37e50813319b2d85164e38d4d

**Comment by santiagohermo** [comment]
Thanks @jmshapir @SimonFreyaldenhoven! Waiting for comments by @nateschor and @ew487 sounds good to me. I just wanted to point out that all tests work without the deleted file.

I implemented your suggestion and re-ran the input folder @jmshapir.

**Review by jmshapir — APPROVED** [review]
Great, thanks @santiagohermo! All seems good to me. If we don't hear an objection in ~48 hours let's go ahead and merge.

**Comment by santiagohermo** [comment]
Summary [here](https://github.com/JMSLab/eventstudyr/issues/25#issuecomment-1479725788)

**Pull request merged**